db/migrate/20221107013442_create_categories.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCategories < ActiveRecord::Migration[7.0]
+  def change
+    create_table :categories do |t|
+      t.string :Title
```

Comment:
It will good if we're naming for variable like "title" instead of "Title". Moreover, we often use camel case in rails

You can search `naming conventions` (Camel Case vs. Snake Case vs. Pascal Cases)